### PR TITLE
Feature gallery listings

### PR DESCRIPTION
**Pull request by johnpooler**

### CODE REVIEW

**Review by nathan-rees — COMMENTED** [review]
do db

File: app/controllers/MoviesGallery.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package controllers
+
+import akka.http.scaladsl.model.HttpHeader.ParsingResult.Ok
+import controllers.movieDatabase
+import javax.inject.Inject
+import play.api._
+import play.api.mvc.{AbstractController, ControllerComponents}
+import play.modules.reactivemongo.{MongoController, ReactiveMongoApi, ReactiveMongoComponents}
+import play.mvc.Controller
```
Comment (RafSobolQAC) [inline app/controllers/MoviesGallery.scala:9]: Codacy Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243893)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243893&bid=17187721)

File: app/controllers/MoviesGallery.scala (hunk 1 above)
Comment (nathan-rees) [reply app/controllers/MoviesGallery.scala:9]: rip

**Pull request closed without merging**